### PR TITLE
Move the project to a pure Apple build

### PR DESCRIPTION
**Pull request by kingst**
This PR includes:

- Remove our dependency on Cocoapods for setting up workspaces
- Includes an app for manual testing and an app for UI testing
- Results in a xcframework, which can be deployed using SPM and looks like it's possible with Cocoapods
- Unit tests + the UI tests from the CardScanSystemTests repo

Before I merge I still need to:

- Update the readme / docs  so that people can quickly get up and running

In subsequent RPs I will:

- Get the XCFramework scripts working
- Deploy the SPM and Cocoapod
- setup our UI tests so that they work on the simulator

### CODE REVIEW

File: CardScanExample/CardScanExample/AppDelegate.swift [hunk 1]
```diff
@@ -16,10 +16,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-        let bundle = CSBundle.bundle()
-        CSBundle.cardScanBundle = bundle
-        CSBundle.bundleName = ""
-        CSBundle.extensionName = ""
+        //let bundle = CSBundle.bundle()
+        //CSBundle.cardScanBundle = bundle
```
Comment (awushensky) [inline CardScanExample/CardScanExample/AppDelegate.swift:20]: Should these be removed?

File: CardScanSystemTest/CardScanSystemTest/ViewController.swift [hunk 2]
```diff
@@ -0,0 +1,377 @@
...
+        self.currentTestImages = self.testImages.compactMap { $0.cgImage }
+        
+        vc.testingImageDataSource = self
+        vc.showDebugImageView = true
+        
+        self.present(vc, animated: true)
+    }
+}
+
+/*
```
Comment (awushensky) [inline CardScanSystemTest/CardScanSystemTest/ViewController.swift:204]: Can we delete this?

File: CardScanSystemTest/CardScanSystemTestUITests/DenyCameraUITests.swift [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        self.interruptionToken.map { removeUIInterruptionMonitor($0) }
+    }
+
+
+    // The way to test this is to run it on a real device with the CardScanSystemTest
+    // app uninstalled so it can get the camera permission dialog and test it
+    /*
+    func testDeny() {
```
Comment (awushensky) [inline CardScanSystemTest/CardScanSystemTestUITests/DenyCameraUITests.swift:38]: And this

File: CardScanSystemTest/CardScanSystemTest/Resources/.gitignore [hunk 4]
```diff
@@ -0,0 +1 @@
+apikey.json
```
Comment (jaimejiyepark) [inline CardScanSystemTest/CardScanSystemTest/Resources/.gitignore:1]: are we handling api keys this way only for systems test?

File: CardScanExample/CardScanExample/AppDelegate.swift (hunk 1 above)
Comment (kingst) [reply CardScanExample/CardScanExample/AppDelegate.swift:20]: yes

File: CardScanSystemTest/CardScanSystemTest/ViewController.swift (hunk 2 above)
Comment (kingst) [reply CardScanSystemTest/CardScanSystemTest/ViewController.swift:204]: yes

File: CardScanSystemTest/CardScanSystemTestUITests/DenyCameraUITests.swift (hunk 3 above)
Comment (kingst) [reply CardScanSystemTest/CardScanSystemTestUITests/DenyCameraUITests.swift:38]: We should keep this one. It's something that I use periodically to test permissions

File: CardScanSystemTest/CardScanSystemTest/Resources/.gitignore (hunk 4 above)
Comment (kingst) [reply CardScanSystemTest/CardScanSystemTest/Resources/.gitignore:1]: yes, only for system test

**Review by awushensky — APPROVED** [review]

**Pull request merged**